efel/cppcore/LibV1.cpp

```diff
@@ -271,7 +272,7 @@ int LibV1::firing_rate(mapStr2intVec& IntFeatureData,
     return nSize;
   else {
     vector<double> stimStart, stimEnd, peakVTime, firing_rate;
-    double lastAPTime;
+    double lastAPTime = 0.;
```

Comment:
We have to avoid NaNs, so it's best to initialise lastAPTime. But if ncount is 0, the product will automatically be 0, no ?